modules/ocf/files/puppet-trigger

```diff
@@ -49,6 +50,20 @@ def switch_to_environment(env):
     )
 
 
+def update_motd(env, reason):
+    dt = datetime.datetime.now()
+    user = os.environ.get('SUDO_USER', 'unknown')
+    if env is None:
+        env = ''
+    with open('/etc/motd', 'a') as f:
+        f.write('[{dt}] Switched to current environment {env} by user {user} for reason: {reason}\n'.format(
```

Comment:
Will this reset when the environment is put on production? Also I don't know, but I was taught updating `/etc/update-motd.d` was better. Though perhaps not in this instance?

Reply:
I believe this will reset on the next puppet run actually, regardless of environment: https://github.com/ocf/puppet/blob/8ec63f78a25c28dfb4f00ba60279dc71bfc1c58e/modules/ocf/manifests/motd.pp#L14

What about writing the reason to a separate file and only displaying it in the motd if it's not on the production env?